gui/encyclopedia/articles/civilizations/sele/military/war_horses.json

```diff
@@ -1,5 +1,5 @@
 {
-    "title": "Seleucid war horses",
+    "title": "Seleucid War Horses",
     "subtitle": "Different mounts from all corners of the empire",
     "content": "When it came to supply of good horses, the vast lands under Seleucid control also harbored various breeds suitable for warfare. Horses native to the Eurasian steppe had a long history of serving as mounts for cavalry. Although small, they were very endurant and thus perfect for hit-and-run tactics. Nisean horses, originating from the Nisean plains in the region of Media in modern-day Iran, on the other hand, were praised for their almost supernatural strength and were by contemporary Chinese authors named “Heavenly Horses”. They were regarded as the perfect breed for heavy cavalry and cataphract warfare.\nIn contrast, Eastern Turanian horses bred by the ancient Turks were taller and slimmer, therefore useful as mounts for light cavalry.\nHorses from the Mediterranean made up the middle ground, neither particularly tall, strong, endurant, or fast, but versatile. They were utilized for rather light to rather heavy cavalry units."
```

Comment:
```suggestion
    "content": "When it came to a supply of good horses, the vast lands under Seleucid control harbored various breeds suitable for warfare. Horses native to the Eurasian steppe had a long history of serving as mounts for cavalry. Although small, they were very enduring and thus perfect for hit-and-run tactics. Nisean horses, originating from the Nisean plains in the region of Media(modern-day Iran) , were praised for their almost supernatural strength and were named "Heavenly Horses" by contemporary Chinese authors . They were regarded as the perfect breed for heavy cavalry and cataphract warfare.\nIn contrast, Eastern Turanian horses bred by the ancient Turks were taller and slimmer, therefore useful as mounts for light cavalry.\nHorses from the Mediterranean made up the middle ground, neither particularly tall, strong, enduring, or fast, but versatile. They were utilized for rather light to rather heavy cavalry units."
```

Reply:
Why are the last paragraphs highlighted in red?

Reply:
I have no idea.

Reply:
Ah, it's because of the quotation marks you added; they cut off the string. Replacing them with curly quotation marks will fix it.

Reply:
That should do it.